src/test/resources/db/prldb.sql

```diff
@@ -169,7 +169,8 @@ COPY public.harvestjobs (id, institutionID, repositoryBaseURL, metadataPrefix, s
 --
 
 ALTER TABLE ONLY public.harvestjobs
-    ADD CONSTRAINT harvestjobs_fkey FOREIGN KEY(institutionID) REFERENCES public.institutions(id);
+    ADD CONSTRAINT harvestjobs_fkey FOREIGN KEY(institutionID) REFERENCES public.institutions(id)
+    ON DELETE CASCADE;;
```

Comment:
Why this change?

Reply:
The `ON DELETE CASCADE` causes linked harvestjob records to be deleted when the "parent" institution record is deleted.  This was a specified requirement for the `removeInstitution` method; doing it in the db by the cascade command is faster than via query.

Reply:
Okay. I assume that without this change, attempting to delete an institution that has jobs associated with it would cause an error? If so, then I think this should have a separate bugfix PR that also adds a test proving that the schedule store service works as expected in that scenario.

Reply:
I agree on the new test. In my experience, without the delete cascade, deleting an institution record would leave orphaned records in the harvetjobs table, which would be an error from our viewpoint but not from the db's viewpoint.

Reply:
Even with the foreign key constraint? That seems odd to me.

Reply:
Anyway... agreed that it's an issue.

Reply:
In addition to it being a separate PR, I would prefer deferring it until after the scheduler service tickets are done, since I am currently in the middle of a major refactor of the schedule store service tests. I think it's fine to let this bug exist until then.